src/utils/generateFlowDiagramData.js

```diff
@@ -97,14 +98,25 @@ const generateFlowDiagramData = (data) => {
     { flowData: {}, mostRecentYear: -Infinity, mostRecentMonth: -Infinity }
   );
 
-  Object.values(flowData).forEach((item) => {
-    const hint = item.isNotAvailable
+  Object.values(flowData).forEach((metric) => {
+    const hint = metric.isNotAvailable
       ? null
-      : generateHint(mostRecentYear, mostRecentMonth, item.item);
+      : generateHint(mostRecentYear, mostRecentMonth, metric.item);
 
     if (hint) {
       // eslint-disable-next-line no-param-reassign
-      item.hint = hint;
+      metric.hint = hint;
+    }
+
+    const { item } = metric;
+
+    const isTooStale = metric.isNotAvailable
```

Comment:
I think this null assignment that never really gets used could be cleaned up a bit. I would change this logic block to something like:

```
if (!metric.isNotAvailable) {
    metric.isTooStale = metric.month < mostRecentMonth && metric.year <= mostRecentYear - 1;
}
```

Reply:
Done